src/main/java/com/example/demo/Dto/response/FullProjectResponseDto.java

```diff
@@ -26,4 +26,14 @@ public class FullProjectResponseDto {
 	private List<DepartmentResponseDto> department;
 	
 	private List<EmployeeResponseDto> employee;
+
+	public FullProjectResponseDto(Long id, String name, LocalDate startDate, LocalDate endDate) {
```

Comment:
Can you give some meaningful name to this class, **FullProjectResponseDto** does not explain the purpose of this class

Reply:
Hi Suraj,

Thank you for the feedback. I understand the need for a more meaningful name. I will rename the class to something more descriptive that better explains its purpose. I think `DetailedProjectResponseDto` will be good for that purpose.